pwa/src/apiService/resources/translation.ts

```diff
@@ -1,29 +1,33 @@
-import { AxiosInstance, AxiosResponse } from "axios";
+import { AxiosInstance, AxiosResponse } from "axios"
 
 export default class Translation {
-  private _instance: AxiosInstance;
+  private _instance: AxiosInstance
 
-  constructor(_instance: AxiosInstance) {
-    this._instance = _instance;
+  constructor (_instance: AxiosInstance) {
+    this._instance = _instance
   }
 
-  public getAllWithTableName = (tableName: string): Promise<AxiosResponse> => {
-    return this._instance.get(`/translations?translationTable=${tableName}`);
-  };
+  public getAllFrom = (tableName: string): Promise<AxiosResponse> => {
+    return this._instance.get(`/translations?translationTable=${tableName}`)
+  }
 
-  public getAllTableNames = (): Promise<AxiosResponse> => {
-    return this._instance.get("/table_names");
-  };
+  public getOne = (id: string): Promise<AxiosResponse> => {
+    return this._instance.get(`/translations/${id}`)
+  }
 
   public create = (data: any): Promise<AxiosResponse> => {
-    return this._instance.post("/translations", JSON.stringify(data));
-  };
+    return this._instance.post('/translations', JSON.stringify(data))
+  }
 
   public update = (data: any, id: string): Promise<AxiosResponse> => {
-    return this._instance.put(`/translations/${id}`, JSON.stringify(data));
-  };
+    return this._instance.put(`/translations/${id}`, JSON.stringify(data))
+  }
 
   public delete = (id: string): Promise<AxiosResponse> => {
     return this._instance.delete(`/translations/${id}`)
   }
+
+  public getTableNames = (): Promise<AxiosResponse> => {
+    return this._instance.get("/table_names")
+  }
```

Comment:
Most changes here are just formatting changes, these happen because you're not using the `.prettierrc` file. Please install the `Prettier` plugin and start using this file. Then, please re-save all these files so that the formatting is the same as the rest of the team.